web/plates/js/radar.js

```diff
@@ -7,8 +7,8 @@ var soundType = 3;  // speech  and sound output, 0=beep+speech (default) 1=Beep
 const showTraces = 1;   // show traces of planes
 // const showTraces = 0;   // do not show trace route  of planes
 
-const cutoff = 20;    // time in seconds how long a plane is displayed after last packet, 
-		      // stratux uses 59 secs in traffic display
+const radarCutoff = 29;    // time in seconds how long a plane is displayed after last packet, 
+		      // keep in sync with cutoff (just below) value in traffic.go and traffic.js 
```

Comment:
Why does it really need to be in sync, as the comment suggests?

Reply:
same as above, otherwise traffic shows target, radar and EFB do not